### DISCUSSION TITLE
carla.Location and get_location() 's difference

### DISCUSSION THREAD

**Discussion by EnjoyingSunshine** [carla-simulator/carla#5459] [Q&A]
carlar.Location function gets the absolute location for map ？
self.get_location gets the relative location for self?

for self.get_location ,have some examples :

bp = blueprint_library.filter('model3')[0]                 
transform = Transform(Location(x=57.403641, y=28.343533, z=0.600000), Rotation(pitch=0, yaw=0, roll=0))
vehicle = world.spawn_actor(bp, transform)  

vehicle.get_location() return :(2 situations)
1.Location(x=00.000000, y=00.000000, z=00.000000)
2.Location(x=57.403641, y=28.343533, z=0.600000)

**Comment by BaoJie456** [comment]
self.get_location returns the absolute location for map. For your example, the result is the second one(Location(x=57.403641, y=28.343533, z=0.600000)).
